modules/oc-mirror-enclave-support.adoc

```diff
@@ -0,0 +1,127 @@
+// Module included in the following assemblies:
+//
+// * installing/disconnected_install/installing-mirroring-disconnected-v2.adoc
+
+:_mod-docs-content-type: Procedure
+[id="oc-mirror-enclave-support_{context}"]
+== How to mirror to an enclave
+
+When you mirror to an enclave, you must first transfer the necessary images from one or more enclaves into the enterprise central registry.
+
+The central registry is situated within a secure network, specifically a disconnected environment, and is not directly linked to the public internet. But the user must execute `oc mirror` in an environment with access to the public internet.
+
+.Procedure
+
+. Before running oc-mirror plugin v2 in the disconnected environment, create a `registries.conf` file.
++
+It is recommended to store the file under `$HOME/.config/containers/registries.conf` or `/etc/containers/registries.conf`.
++
+The TOML format of the file is described in this specification.
++
+.Example `registries.conf`
+[source,toml]
+----
+[[registry]]
+location="registry.redhat.io"
+[[registry.mirror]]
+location="<enterprise-registry.in>"
+
+[[registry]]
+location="quay.io"
+[[registry.mirror]]
+location="<enterprise-registry.in>"
+----
+
+. Generate a mirror archive.
+
+.. To collect all the {product-title} content into an archive on the disk under `<file_path>/enterprise-content`, run the following command:
++
+[source,terminal]
+----
+$ oc mirror --v2 -c isc.yaml file://<file_path>/enterprise-content
+----
++
+.Example of isc.yaml:
+[source,yaml]
+----
+apiVersion: mirror.openshift.io/v2alpha1
+kind: ImageSetConfiguration
+mirror:
+  platform:
+    architectures:
+      - "amd64"
+    channels:
+      - name: stable-4.15
+        minVersion: 4.15.0
+        maxVersion: 4.15.3
+----
++
+After the archive is generated, it is transferred to the disconnected environment. The transport mechanism is not part of oc-mirror plugin v2. The enterprise network administrators determine the transfer strategy.
++
+In some cases, the transfer is done manually, in that the disk is physically unplugged from one location, and plugged to another computer in the disconnected environment. In other cases, the Secure File Transfer Protocol (SFTP) or other protocols are used.
+
+. After the transfer of the archive is done, you can execute oc-mirror plugin v2 again in order to mirror the relevant archive contents to the registry (`entrerpise_registry.in` in the example). 
++
+[source,terminal]
+----
+$ oc mirror --v2 -c isc.yaml --from file://<disconnected_environment_file_path>/enterprise-content docker://<enterprise_registry.in>/
+----
+Where:
+- `--from` points to the folder containing the archive. It starts with the `file://`.
+- The destination of the mirroring is the final argument. Because it is a docker registry, it is prefixed by `docker://`.
```

Comment:
```suggestion
- The destination of the mirroring is the final argument. Because it is a docker registry, it is prefixed by `docker://`.
```

Missing the target tag. For example `--from`